### PR TITLE
Added new operators: `in`, `in caseless`, `not in` and `not in caseless`.

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
- Fixed an issue with causing `Circular reference detected` with using `is updated with`.
- Fixed an issuw that `contains` doesn't work for non-string values.
- Added new operators: `in`, `in caseless`, `not in` and `not in caseless`.
- Updated the docker version: `demisto/python3:3.9.5.21272`

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ExtFilter/ReleaseNotes/1_1_4.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+##### ExtFilter
+- Fixed an issue with causing `Circular reference detected` with using `is updated with`.
+- Fixed an issuw that `contains` doesn't work for non-string values.
```
Comment (Noy-Maimon) [inline Packs/ExtFilter/ReleaseNotes/1_1_4.md:5]:
```suggestion
- Fixed an issue where the 'contains' opertator would not work for non-string values.
```

File: Packs/ExtFilter/ReleaseNotes/1_1_4.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+##### ExtFilter
+- Fixed an issue with causing `Circular reference detected` with using `is updated with`.
```
Comment (Noy-Maimon) [inline Packs/ExtFilter/ReleaseNotes/1_1_4.md:4]:
```suggestion
- Fixed an issue where the `is updated with` operator was causing circular references. 
```

**Review by Noy-Maimon — APPROVED** [review]

**Comment by Noy-Maimon** [comment]
@spearmin10 Thank you for your contribution!
Please see the suggested changes I commented on the release notes file.
Plus, please sign our [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) in order to proceed the contribution process.

**Comment by spearmin10** [comment]
> @spearmin10 Thank you for your contribution!
> Please see the suggested changes I commented on the release notes file.
> Plus, please sign our [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) in order to proceed the contribution process.

Thanks, I've updated the release note following to your comments.

**Pull request merged**